e2e/autoscript-apps/src/authz-txn-basic-redirect/autoscript.ts

```diff
@@ -39,10 +39,7 @@ function autoscript() {
             }
             break;
           case 'AUTHENTICATE':
-            if (
-              action.payload.type === 'composite_advice' &&
-              typeof action.payload.tree === 'string'
-            ) {
+            if (action.payload.tree === '') {
```

Comment:
This is necessary for the tests to pass, now that we are only passing `authIndexType=composite_advice` (or indeed any query parameters at all), on the initial authentication request when we act upon the advice returned by the server. An action type of `AUTHENTICATE` and no tree specified is about as specific as I can be here, I think, when determining which stage of the test we are at for the suite.